media/base/decoder_buffer.cc

```diff
@@ -47,7 +72,13 @@ DecoderBuffer::DecoderBuffer(const uint8_t* data,
 }
 
 DecoderBuffer::DecoderBuffer(std::unique_ptr<uint8_t[]> data, size_t size)
+#if !BUILDFLAG(IS_COBALT)
```

Comment:
Should it also be guarded by `USE_STARBOARD_MEDIA`?

Reply:
This is original Chrome code, and we don't use it.